ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -138,6 +146,10 @@ message GetTransactionByEventIdRequest {
   // Each element must be a valid PartyIdString (as described in ``value.proto``).
   // Required
   repeated string requesting_parties = 3;
+
+  // Must be a valid domain ID
+  // Required
+  string domain_id = 4;
 }
 
 message GetTransactionByIdRequest {
```

Comment:
We probably also want corresponing endpoinds for retrieving individual transfers by their transfer ID, don't we?

Reply:
This is a spot on question! I think this is related to the TODO I've added to the top if this file:
```
// TODO adding transfer transactions to API read side could be done differently. Some ideas:
//   - integrating on the event level, and transparent above (very small diff to 2.0, but Transaction has more info than
//   it would make sense for transfers, and TransactionTree/Transaction will be a little odd with the singular event in them)
//   - keeping "transactions" in API nomenclature separate from "transfers" completely and defining separate streaming/
//   pointwise endpoints for this new purpose (clean nomenclature and separation, but need to synchronize more at client side)
```
The current approach in the PR is that we integrate at the response container level (see TransactionTreeOrTransfer, TransactionOrTransfer), so we still see merged daml transactions/transfer transactions on the transaction service endpoints, but integrate as high as possible so it is less ambiguous on the data-DTO level.
Getting back to your question: since this integration happens below, we do not need another endpoint, since we can query with this request transfers too. With following your suggestion I think we might be going for the alternative approach I am suggesting in the TODO (the second one): to have completely separate endpoints for transfers and daml transactions. This is also very well possible.

Reply:
My question is not about this. For transactions, we have endpoints for querying by transaction ID. I think we want something similar for transfers, independent of whether there's a single stream or multiple streams.

> to have completely separate endpoints for transfers and daml transactions. This is also very well possible.

I very much like the unified stream in the current proposal.

Reply:
As discussed we introduced Update-s to capture the unified stream of transactions and transfers what the transaction service became.